core.js

```diff
@@ -404,9 +388,6 @@
             }
         }
 
-        /* build:app */
-        initDebug();
```

Comment:
@stepek There is some inconsistency, because this basically makes the `debug` module useless.

I'm not sure if this module still works correctly and can't check it right now, but
- If it does, I think these lines should not be removed
- If it doesn't, the debug module should be removed (looks like this would be consistent with the version in 2.3.1, but then we would lose this functionality)